### PR TITLE
tlg0555.tlg001-tlg0555.tlg007

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated the cts.xml files to current standards
Updated the bibliographic metadata as well

### CODE REVIEW

File: data/tlg0555/tlg001/tlg0555.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -24,6 +24,10 @@
                 <pubPlace>Leipzig</pubPlace>
                 <date>1905</date></imprint>
             </monogr>
+           <series>
```
Comment (lcerrato) [inline data/tlg0555/tlg001/tlg0555.tlg001.opp-grc1.xml:27]: `<title>Protrepticus und Paedagogus</title>` (above) should probably have a "mul" language attribute

File: data/tlg0555/tlg002/tlg0555.tlg002.opp-grc1.xml [hunk 2]
```diff
@@ -24,8 +24,12 @@
 							<publisher>Hinrichs</publisher>
 							<pubPlace>Leipzig</pubPlace>
 							<date>1905</date>
```
Comment (lcerrato) [inline data/tlg0555/tlg002/tlg0555.tlg002.opp-grc1.xml:26]: see previous comment

File: data/tlg0555/tlg004b/tlg0555.tlg004b.opp-grc1.xml [hunk 3]
```diff
@@ -18,15 +18,19 @@
 				<biblStruct>
 					<monogr>
 						<author>Clement of Alexandria</author>
-						<editor role="editor">Otto Stählin</editor>
-						<title>Clemens Alexandrinus Dritter Band</title>
+						<editor>Otto Stählin</editor>
+						<title>Stromata, Buch VII-VIII. Excerpta ex Theodoto. Eclogae prophetica.</title>
```
Comment (lcerrato) [inline data/tlg0555/tlg004b/tlg0555.tlg004b.opp-grc1.xml:22]: should probably have a "mul" language attribute as it is German and Latin.

File: data/tlg0555/tlg005/tlg0555.tlg005.opp-grc1.xml [hunk 4]
```diff
@@ -18,15 +18,19 @@
 				<biblStruct>
 					<monogr>
 						<author>Clement of Alexandria</author>
-						<editor>Otto Stahlin</editor>
-						<title xml:lang="lat">Clemens Alexandrinus</title>
+						<editor>Otto Stählin</editor>
+						<title>Stromata, Buch VII-VIII. Excerpta ex Theodoto. Eclogae prophetica.</title>
```
Comment (lcerrato) [inline data/tlg0555/tlg005/tlg0555.tlg005.opp-grc1.xml:22]: same

File: data/tlg0555/tlg006/__cts__.xml [hunk 5]
```diff
@@ -2,6 +2,6 @@
   <ti:title xml:lang="lat">Quis dives salvetur</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0555.tlg006.opp-grc1" workUrn="urn:cts:greekLit:tlg0555.tlg006">
     <ti:label xml:lang="lat">Quis dives salvetur</ti:label>
-    <ti:description xml:lang="lat">Quis dives salvetur, Clement of Alexandria, Stählin, 1909</ti:description>
+    <ti:description xml:lang="lat">Clement of Alexandria.  Clemens Alexandrinus, Volume 3. Stählin, Otto, editor. Leipzig: Hinrichs, 1909.</ti:description>
```
Comment (lcerrato) [inline data/tlg0555/tlg006/__cts__.xml:5]: change language from "lat" to "mul"

File: data/tlg0555/tlg006/tlg0555.tlg006.opp-grc1.xml [hunk 6]
```diff
@@ -19,14 +19,18 @@
 					<monogr>
 						<author>Clement of Alexandria</author>
 						<editor>Otto Stählin</editor>
-						<title xml:lang="lat">Clemens Alexandrinus</title>
+						<title>Stromata, Buch VII-VIII. Excerpta ex Theodoto. Eclogae prophetica.</title>
```
Comment (lcerrato) [inline data/tlg0555/tlg006/tlg0555.tlg006.opp-grc1.xml:22]: same

File: data/tlg0555/tlg006/tlg0555.tlg006.opp-grc1.xml [hunk 7]
```diff
@@ -19,14 +19,18 @@
...
 						<imprint>
 							<publisher>Hinrichs</publisher>
 							<pubPlace>Leipzig</pubPlace>
 							<date>1909</date>
 						</imprint>
-						<biblScope unit="volume">3</biblScope>
 					</monogr>
+					<series>
+						<title level="s">Clemens Alexandrinus
```
Comment (lcerrato) [inline data/tlg0555/tlg006/tlg0555.tlg006.opp-grc1.xml:30]: should be "lat" title attribute

**Review by lcerrato — CHANGES_REQUESTED** [review]
`<title>Protrepticus und Paedagogus</title>` should probably have a "mul" language attribute

**Comment by lcerrato** [comment]
@AlisonBabeu 
I would just double check the titles in the files — most appear to be either a combination of Latin and German or Latin, so we probably want some language attributes there.  I don't think I flagged them all because the edited portions of the files weren't always visible.
One of the metadata files had a "lat" where a "mul" belonged, but the editions are where I spotted the missing info.

**Comment by AlisonBabeu** [comment]
hi @lcerrato I will make those changes! Thanks for reviewing!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**